CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# CONTRIBUTING
+
+## Style Guide
+
+In the following templates, `TYPE` may be any of `Fix`, `Feature`, `Task`, or `Enhancement`.
```

Comment:
Also, should it be Bug instead of Fix? Fix describes the PR, but Bug describes the ticket/issue... meh, language.